From b7089121e305ef29dee9b72bfdb8b1496ffed48c Mon Sep 17 00:00:00 2001
From: Ben Finney <ben+python@benfinney.id.au>
Date: Apr 04 2019 08:05:19 +0000
Subject: Create the socket and catch “non-socket” errors.


---

diff --git a/daemon/daemon.py b/daemon/daemon.py
index 1a75bc4..df2d3a8 100644
--- a/daemon/daemon.py
+++ b/daemon/daemon.py
@@ -761,9 +761,8 @@ def is_socket(fd):
         """
     result = False
 
-    file_socket = socket.fromfd(fd, socket.AF_INET, socket.SOCK_RAW)
-
     try:
+        file_socket = socket.fromfd(fd, socket.AF_INET, socket.SOCK_RAW)
         file_socket.getsockopt(socket.SOL_SOCKET, socket.SO_TYPE)
     except socket.error as exc:
         exc_errno = exc.args[0]

