--- setup.py.orig	2010-01-08 20:29:20.000000000 -0500
+++ setup.py	2010-01-08 20:31:00.000000000 -0500
@@ -19,7 +19,7 @@
 
 distribution_name = "python-daemon"
 main_module_name = 'daemon'
-main_module = __import__(main_module_name, fromlist=['version'])
+main_module = __import__(main_module_name, globals(), {}, ['version'])
 version = main_module.version
 
 short_description, long_description = (
